sale_blanket_order_line_position/models/blanket_orders.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class BlanketOrder(models.Model):
+    _name = "sale.blanket.order"
+    _inherit = ["sale.blanket.order", "order.position.mixin"]
+
+    def recompute_positions(self):
```

Comment:
same thing here.
I feel like it could be done in the mixin.

Reply:
Also, I would have added an `@api.depends("")` here, and dropped the `ir.action.report` override.
```python
@api.depends(
    "locked_positions",
    "company_id.disable_sale_position_recompute",
    "line_ids.display_type",
    "line_ids.sequence",
)
```

Reply:
It can stay on on the mixin if you pass the lines.
The `depends` won't work. Each model has its own field... "order_line", "line_ids", etc.
Also, is more reliable if we call it when/where we need it.